packages/util-utf8/src/toUtf8.browser.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { toUtf8 } from "./toUtf8";
```

Comment:
Explicitly call out jsdom environment in browser tests
```suggestion
/**
 * @jest-environment jsdom
 */
import { toUtf8 } from "./toUtf8";
```

Reply:
fixed